tests/unit/test-utils.js

```diff
@@ -0,0 +1,10 @@
+import Rx from 'rxjs';
+
+export function waitFor(fn) {
```

Comment:
I don't have any opinion either. Let's see how this works for us as we add more tests.